src/app/tests/suites/ciTests.json

```diff
@@ -301,6 +301,28 @@
         "Test_TC_ZEOFREMON_2_1"
     ],
     "AirQuality": ["Test_TC_AIRQUAL_1_1", "Test_TC_AIRQUAL_2_1"],
+    "ConcentrationMeasurement": [
```

Comment:
Is one set not enough to run? They are all fundamentally the same.

Reply:
I agree with you although this is what CSG requested. I can discuss this with them next week in Boston.

Reply:
Do we not need to have a run for each cluster as as the features may be different for each one?